db/migrate/20230818114648_devise_create_users.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :first_name,         null: false
+      t.string :last_name,          null: false
```

Comment:
One tip I'd give, and it may not apply so much to GDS work as I don't know if guidance requires first and last designations, is that in non-English languages the family name might be given first. I usually go with "given name" and "family name" when creating a users table. Probably not very relevant to this project though